src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -194,7 +199,7 @@ UCS_PROFILE_FUNC(ucs_status_t, uct_cuda_copy_mem_dereg,
         return UCS_OK;
     }
 
-    status = UCT_CUDA_CALL_LOG_ERR(cudaHostUnregister, address);
+    status = UCT_CUDADRV_FUNC_LOG_ERR(cuMemHostUnregister(address));
```

Comment:
@rakhmets Do we need to handle the case of attempting unregister when there is no active context?